src/components/books/AddBook.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-const-assign */
```

Comment:
- [ ] Let's not disable the linter rules. These are here to help us write clean code.